include/hip/devicelib/single_precision/sp_intrinsics.hh

```diff
@@ -24,6 +24,17 @@
 #define HIP_INCLUDE_DEVICELIB_SP_INTRINSICS_H
 
 #include <hip/devicelib/macros.hh>
+#include <hip/devicelib/single_precision/sp_math.hh>
+
+static inline __device__ float __expf(float x) { return ::expf(x); }
+
+static inline __device__ float __logf(float x) { return ::log(x); }
+static inline __device__ float __log2f(float x) { return ::log2(x); }
+static inline __device__ float __log10f(float x) { return ::log10(x); }
```

Comment:
why not `return std::log10(x);`

Reply:
To me it does not seem to matter which one we call here.